src/io/mod.rs

```diff
@@ -11,11 +11,10 @@ use serde::{Deserialize, Serialize};
 use thiserror::Error;
 
 mod stun;
-pub use stun::StunMessage;
 pub(crate) use stun::{
-    Class as StunClass, Method as StunMethod, StunError, StunTiming, TransId,
-    DEFAULT_MAX_RETRANSMITS,
+    Class as StunClass, Method as StunMethod, StunError, StunTiming, DEFAULT_MAX_RETRANSMITS,
 };
+pub use stun::{StunMessage, StunMessageBuilder, TransId as StunTransId};
```

Comment:
I don't think we need this renaming of TransId -> StunTransId. It causes unnecessary code churn.

Reply:
My thinking was that all the other exports are prefixed Stun so for consistency it should also have the prefix. However, we could keep just `TransId` internally

Reply:
Let's keep it for now and make a separate PR to address that.